test/common/testvm.py

```diff
@@ -107,6 +107,7 @@ def __init__(self, address=None, image=None, verbose=False, label=None, fetch=Tr
         self.arch = "x86_64"
 
         self.image = image or testinfra.DEFAULT_IMAGE
+        self.atomic_image = self.image in ["rhel-atomic", "fedora-atomic", "continuous-atomic"]
```

Comment:
The problem is that many of the cases where ```m.atomic.image``` is used in this pull request should not be consistent in how they check this. The various Atomic's gain features at different rates.